README.md

```diff
@@ -1 +1,67 @@
-# awesome-wonderland-engine
+# Awesome Wonderland Engine [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+[<img src="https://yt3.googleusercontent.com/eZZOHhhAotLIcCXPNJ78lMrW-4wtrLJAg5ZqiEuqHkDMNKbBk3ZIHdVmfTdGEHlxmNcDrQVWFQ8=s900-c-k-c0x00ffffff-no-rj" align="right" width="100">](https://immersiveweb.dev/)
+
+> A curated list of projects made with wonderland engine.
+
+[Wonderland Engine](https://wonderlandengine.com/) is a highly performant, accessible, and lightweight graphics engine tailored for the web and [WebXR](https://immersiveweb.dev/). It comes with a desktop editor application and an efficient WebAssembly-based runtime.
+
+## Contents
+
+-   [Awesome Wonderland Engine ](#awesome-wonderland-engine-)
+    -   [Contents](#contents)
+    -   [Official Resources](#official-resources)
+    -   [Integrations](#integrations)
+    -   [Featured Projects](#featured-projects)
+    -   [Community Projects](#community-projects)
+    -   [Contribute](#contribute)
+
+## Official Resources
+
+-   [Documentation](http://example.com)
+-   [Written tutorials](https://wonderlandengine.com/tutorials/)
+-   [Youtube resources](https://www.youtube.com/c/WonderlandEngine)
+-   [Examples](https://github.com/WonderlandEngine/wonderland-engine-examples)
+-   [Cloud Examples](https://github.com/WonderlandEngine/wonderland-cloud-examples)
+-   [Components](https://github.com/WonderlandEngine/components)
+-   [Plugins](https://github.com/WonderlandEngine/plugins)
+-   [Screenshot testing](https://github.com/WonderlandEngine/wonderland-screenshot-testing)
+
+## Integrations
+
+-   [React UI](https://github.com/WonderlandEngine/wonderland-react-ui)
```

Comment:
I would move this to a separate UI libraries section and add Canvas UI from Nik Lever and LazyWidgets from Playko